usr/sbin/p11sak/p11sak.1

```diff
@@ -0,0 +1,46 @@
+.\" Copyright IBM Corp. 2020
```

Comment:
I think the man page belongs into <ock>/man/man1/, not into the tools directory.

Reply:
And also add it to man1.mk so that it gets installed during make install

Reply:
this is resolved.